src/tslint-cli.ts

```diff
@@ -121,7 +121,7 @@ const options: Option[] = [
     {
         short: "r",
         name: "rules-dir",
-        type: "string",
+        type: "array",
```

Comment:
isn't it `string | string[]`? can we support both types in the schema?

Reply:
@giladgray 
Not sure i've understood correctly, but

These are `cli` option definitions. The `type` property is handled by the [cli code](https://github.com/ggarek/tslint/blob/f3f4f3dda47b554d5ba7ab34e403ae0ab1a589f6/src/tslint-cli.ts#L213) basically adding `collect` coercion to commander. _(and we can not actually use `string | string[]` here, these are typescript types, but we have just value of type `type: "string" | "boolean" | "array";`)_

> It seems like the codebase is ready for both string and string[] for rulesDirectory, hence the change is just a cli option creation. (I have traced it back to Linter > getEnabledRules > arrayify)

Considering [this](https://github.com/ggarek/tslint/blob/f3f4f3dda47b554d5ba7ab34e403ae0ab1a589f6/src/linter.ts#L238), CLI should be ready to handle both `string` and `string[]` values.